OpenEdXMobile/src/main/java/org/edx/mobile/base/FindCoursesBaseActivity.java

```diff
@@ -110,6 +110,7 @@ private void setupWebView() {
 
         client.setActionListener(this);
         client.setPageStatusListener(this);
+        webview.getSettings().setDomStorageEnabled(true);
```

Comment:
Should this be enabled in all web views? If so, then this could be moved to the initialization config in `EdxWebView`. Otherwise maybe move this to the start of the method separate from the web view client configuration?